### PR TITLE
Math-magician with with hooks

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is my Math App project. this component was dynamically built with the following:

- npx create-react-app
- HTML
- CSS
- JavaScript
- Refractor code with hooks

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,54 +1,49 @@
...
-  handleClick = (event) => {
-    const buttonName = event.target.innerText;
-    const result = calculate(this.state, buttonName);
-    this.setState(result);
-  }
+function handleClick(event, state, setter) {
+  const buttonName = event.target.innerText;
+  const result = calculate(state, buttonName);
+  setter(result);
+}
```
Comment (mikethreels) [inline src/components/calculator.js]: - [ ] please do not take the handleClick event out of the component.

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,54 +1,49 @@
...
-            <button type="submit" onClick={this.handleClick} className="box-color">-</button>
-            <button type="submit" onClick={this.handleClick}>1</button>
-            <button type="submit" onClick={this.handleClick}>2</button>
-            <button type="submit" onClick={this.handleClick}>3</button>
-            <button type="submit" onClick={this.handleClick} className="box-color">+</button>
-            <button type="submit" onClick={this.handleClick} className="zero">0</button>
-            <button type="submit" onClick={this.handleClick}>.</button>
-            <button type="submit" onClick={this.handleClick} className="box-color">=</button>
-          </div>
+const CalcUserInterface =() => { // eslint-disable-line
```
Comment (mikethreels) [inline src/components/calculator.js]: - [ ] please do not disable the linters they help us to make our code better.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Amaka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @lawrahkonwea, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job working on the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**